src/test/java/com/nephren/raven/apiclient/serviceExample/client/fallback/POSTExampleClientFallback.java

```diff
@@ -19,4 +22,38 @@ public Mono<ResponseEntity<ServerResponseBody>> postRequest(ServerRequestBody re
     serverResponseBody.setMessage("Fallback during calling postRequest");
     return Mono.just(ResponseEntity.ok(serverResponseBody));
   }
+
+  @Override
+  public Mono<ResponseEntity<ServerResponseBody>> postRequestMultipart(FilePart file) {
+    return null;
+  }
+
+  @Override
+  public Mono<ResponseEntity<ServerResponseBody>> postRequestMultipartNoBody() {
+    return null;
+  }
+
+  @Override
+  public Mono<ResponseEntity<ServerResponseBody>> postRequestMultipartReactiveFlux(
+      Flux<FilePart> file) {
+    return null;
+  }
+
+  @Override
+  public Mono<ResponseEntity<ServerResponseBody>> postRequestMultipartReactiveMono(
+      Mono<FilePart> file) {
+    return null;
+  }
```

Comment:
no need to handle the fallabck, because this file is inside the test folder, and if the test is not intended to failing, it is no need to waste time for handling the fallback properly